src/pages/home/report/ReportActionCompose.js

```diff
@@ -134,13 +138,15 @@ class ReportActionCompose extends React.Component {
 
         this.state = {
             isFocused: this.shouldFocusInputOnScreenFocus,
+            isFullComposerAvailable: props.isComposerFullSize,
             textInputShouldClear: false,
             isCommentEmpty: props.comment.length === 0,
             isMenuVisible: false,
             selection: {
                 start: props.comment.length,
                 end: props.comment.length,
             },
+            maxLines: props.isSmallScreenWidth ? 6 : 16,
```

Comment:
Should we make constants for these?

Reply:
Yeah that sounds like a good idea, I'll add that.